src/component/Restaurants.js

```diff
@@ -1,35 +1,57 @@
 import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { RestaurantListItem } from "./RestaurantListItem";
 
-const api = "/cities";
+import thumbnail1Src from "place-my-order-assets/images/1-thumbnail.jpg";
+import thumbnail2Src from "place-my-order-assets/images/2-thumbnail.jpg";
+import thumbnail3Src from "place-my-order-assets/images/3-thumbnail.jpg";
+import thumbnail4Src from "place-my-order-assets/images/4-thumbnail.jpg";
 
 /**
  * Displays information about available restaurants.
  * @returns {JSX.Element}
  */
 function Restaurants (props) {
-    const [region, setRegion] = useState("");
     const [cities, setCities] = useState({});
-    const [city, setCity] = useState("");
+    const [city, setCity] = useState(DEFAULT_SELECT_VALUE);
+    const [region, setRegion] = useState(DEFAULT_SELECT_VALUE);
+    /** @type {[RestaurantRegionData, (value: RestaurantRegionData) => void]} */
+    const [restaurants, setRestaurants] = useState({});
 
```

Comment:
I feel like all these handlers are getting very complicated. Couldn't this be simplified with some `useEffects`?

Reply:
Or `useReducer`

Reply:
changed